src/common/directives/cd-ga-track-exit-nav.js

```diff
@@ -0,0 +1,14 @@
+import Vue from 'vue';
+
+export default {
+  bind(el) {
+    el.addEventListener('click', (e) => {
+      e.preventDefault();
```

Comment:
This breaks middle/ctrl click, right?

Reply:
It does, indeed. I can not prevent the event, but I'm not sure how to interpret the sendBeacon from MDN. From the content, it should carry over, but I'm expecting ga to polyfill it with xhr for IE11 (https://caniuse.com/#feat=beacon), which requires then the callback